version.go

```diff
@@ -0,0 +1,13 @@
+package meilisearch
+
+import "fmt"
+
+const VERSION = "0.19.1"
+
+func GetQualifiedVersion() (qualifiedVersion string) {
+	return getQualifiedVersion(VERSION)
+}
+
+func getQualifiedVersion(version string) (qualifiedVersion string) {
+	return fmt.Sprintf("Meilisearch Go (v%s)", version)
+}
```

Comment:
What is the point of writing to function instead of one? Is it for having a public and a private one?

Reply:
Yeah, I could add just one function but I was thinking about how to test it, so I created two functions one with the possibility to add a version (private) and another used in the app (public).

This way I could test the format and I could guarantee that the user will always send the right value "VERSION" to the function.
Because I could let this responsibility to the caller:

```go
	request.Header.Set("User-Agent", GetQualifiedVersion(VERSION))
```

But how can I ensure by tests the user is using the `VERSION` and not something else?

PS: I still need to understand more about the Go concepts to improve this 😅, feel free to point a better solution :)